packages/paste-website/src/component-examples/FormPillGroup.tsx

```diff
@@ -92,3 +92,35 @@ render(
   <RemovableFormPillGroup />
 )
 `.trim();
+
+export const i18nExample = `
+const I18nFormPillGroup = () => {
+  const pillState = useFormPillState();
+
+  return (
+    <form>
+      <FormPillGroup
+        {...pillState}
+        aria-label="Votre sports favoris:"
+        i18nKeyboardControls="Appuyez sur Supprimer ou Retour arrière pour supprimer. Appuyez sur Entrée pour basculer la sélection."
+      >
+        <FormPill {...pillState}>
+          <CalendarIcon decorative size="sizeIcon10" />
+          Le tennis
+        </FormPill>
+        <FormPill {...pillState} selected>
+          Le football américain
+        </FormPill>
+        <FormPill {...pillState} selected>
```

Comment:
suggestion: For this example, I wouldn't have some `selected` pills and some not. I tried to click on them instinctively and thought it could be a bug before I realized that it's just how we coded it. I think that unless we make them "selectable" w/ a state, they just just all be not selected.

Reply:
Ohhh I didn't think about that, will update!